testing/full-piglit-run.sh

```diff
@@ -517,17 +551,20 @@ Options:
   --piglit-results-path <path>     <path> to the piglit results
   --vk-gl-cts-path <path>          <path> to the built vk-gl-cts binaries
   --deqp-path <path>               <path> to the built dEQP binaries
+  --crucible-path <path>           <path> to the built crucible binaries
   --vk-cts-prefix <prefix>         <prefix> to use with the vk-cts run
   --gl-cts-prefix <prefix>         <prefix> to use with the gl-cts run
   --deqp-gles2-prefix <prefix>     <prefix> to use with the dEQP GLES2 run
   --deqp-gles3-prefix <prefix>     <prefix> to use with the dEQP GLES3 run
-  --deqp-gles31-prefix <prefix>    <prefix> to use with the dEQP GLES3.1 run
+  --deqp-gles31-prefix <prefix>    <prefix> to use with the crucible run
```

Comment:
You have swapped the documentation incorrectly. Please fix this.

Additionally, you can just force pull the commit and have 1 single commit, and not 2 (or 3).